docs/install/install-local-migrate.md

```diff
@@ -87,89 +86,65 @@ Prior to attempting this step, do consider the following:
 
 While this will depend on your pre-existing Production system, it is important to double-check this. If you have a larger hash size than the default ISLE system (##), and don't follow the settings below, ISLE may not function properly when your data has been migrated.
 
-*  If you have larger Fedora collections, there is a possibility that you made changes to the `akubra-llstore.xml` file to allow for the creation of larger or deeper hash directories)
-  * You will need to copy your `/usr/local/fedora/server/config/spring/akubra-llstore.xml` from your Production Fedora System to `./config/fedora/akubra-llstore.xml`
+*  If you have larger Fedora collections, there is a possibility that you made changes to the `akubra-llstore.xml` file to allow for the creation of larger or deeper hash directories)  
+
+  * You will need to copy your `/usr/local/fedora/server/config/spring/akubra-llstore.xml` from your Production Fedora System to `./config/fedora/akubra-llstore.xml`  
+
   * You will then need to add an extra line in the Fedora service (fedora) volumes section to bind mount this file in. This will guarantee proper Fedora data hash structure.
     * `- ./config/fedora/akubra-llstore.xml:/usr/local/fedora/server/config/spring/akubra-llstore.xml`
 
-#### Solr schema & Islandora transforms
+#### Solr Schema & Islandora Transforms
 
 This data can be challenging depending on the level of customizations to contend with and as such, ISLE maintainers recommends following one of the three strategies outlined below.
 
-##### (**Easy**) - Run "stock" ISLE
+##### **Easy** - Run "Stock" ISLE
```

Comment:
Can we make these sections stick out more?   Ie, indent the material under each section?   I know it says there are three options, but the options don't stand out and might be hard to read.  Maybe this is a limitation of Markdown.

Reply:
Agreed.  I can experiment with that if you like.  Since we say "three options", I think the sections should be "numbered" too.